### PR TITLE
[Android]Authentication using facebook in android studio

### PR DESCRIPTION
**Pull request by Owallah**
This fixes issue #3493

### CODE REVIEW

**Comment by robert-muriithi** [comment]
@hectorkambow @ahmadmardeni1  I am kindly requesting to review this article now that i am done with my PR 🚀

**Comment by ahmadmardeni1** [comment]
Hey, @robert-muriithi Are you a test peer reviewer? 
I can't find his name on the list. @hectorkambow

**Comment by robert-muriithi** [comment]
> Hey, @robert-muriithi Are you a test peer reviewer? I can't find his name on the list. @hectorkambow

No, i am not a peer reviewer but i'd love to be one
@ahmadmardeni1  @hectorkambow

**Comment by WanjaMIKE** [comment]
Unfortunately, the peer review program is closed for now. But we will consider request at some point in the future. Thanks

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

**Comment by Owallah** [comment]
Hello @WanjaMIKE

**Comment by WanjaMIKE** [comment]
@Owallah No worries. The PR is been worked on. Thanks

**Comment by Owallah** [comment]
@WanjaMIKE it's okay. Thanks for the feedback

File: content/articles/authentication-using-facebook-in-android-studio/index.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+ android:layout_height="wrap_content"
+ android:layout_marginTop="24dp"
+ app:layout_constraintEnd_toEndOf="parent"
+ app:layout_constraintStart_toStartOf="parent"
+ app:layout_constraintTop_toBottomOf="@+id/textView5" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
+```
+
+### Step 9: The Code
```
Comment (WanjaMIKE) [inline content/articles/authentication-using-facebook-in-android-studio/index.md:221]: What is happening in the code below. Could you explain it?

File: content/articles/authentication-using-facebook-in-android-studio/index.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        binding.userEmail.text = "Email: ${email}"
+        binding.userBDay.text = "Birthday: ${birthday}"
+        binding.userGender.text = "Gender: ${gender}"
+}
+     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        callBackManager.onActivityResult(requestCode, resultCode, data) }
+}
+```
+
```
Comment (WanjaMIKE) [inline content/articles/authentication-using-facebook-in-android-studio/index.md:292]: Explain your code rather than just pasting it in the article

**Review by WanjaMIKE — CHANGES_REQUESTED** [review]

File: content/articles/authentication-using-facebook-in-android-studio/index.md (hunk 2 above)
Comment (Owallah) [reply content/articles/authentication-using-facebook-in-android-studio/index.md:292]: Okay sure. Let me add the explanation

**Comment by Owallah** [comment]
Hello, @WanjaMIKE 
I have added some explanatory comments.
I'd love to hear your feedback on the same

**Comment by hectorkambow** [comment]
@Owallah 

Please be sure you are going over the entire article for any other mistakes or errors, as our peer reviewers are not responsible to point out every mistake - that is the authors responsibility to properly review and proofread their work.

File: content/articles/authentication-using-facebook-in-android-studio/index.md (hunk 2 above)
Comment (Owallah) [reply content/articles/authentication-using-facebook-in-android-studio/index.md:307]: I have added the comments as asked

**Review by Owallah — COMMENTED** [review]
I have added comments to my lines of code as asked

**Comment by WanjaMIKE** [comment]
Finishing up

**Comment by WanjaMIKE** [comment]
@Owallah Please check the numerous edits I have made across the article. Thanks

**Review by Owallah — COMMENTED** [review]
The changes are okay. Thanks.

**Comment by hectorkambow** [comment]
> @Owallah Please check the numerous edits I have made across the article. Thanks

every article case is unique - but changes should be not done by the reviewer. 

@WanjaMIKE

**Comment by WanjaMIKE** [comment]
@ahmadmardeni1 could you publish this PR. Thanks

**Comment by ahmadmardeni1** [comment]
Great work @Owallah @WanjaMIKE

**Comment by Owallah** [comment]
Thank you so much

**Pull request merged**